src/foam/u2/borders/CollapseBorder.js

```diff
@@ -9,20 +9,17 @@ foam.CLASS({
   name: 'CollapseBorder',
   extends: 'foam.u2.Controller',
 
-  requires: [ 'foam.u2.ActionView' ],
+  imports: [ 'theme' ],
```

Comment:
Should be optional otherwise it will break the border outside nanos

Reply:
Yes, non-nanos demo apps keep getting broken by non-optional imports and I have to keep fixing them.